argyrogounari/assignment1/SinglyLinkedList.java

```diff
@@ -0,0 +1,48 @@
+
+public class SinglyLinkedList<T> {
+
+	private Node<T> head;
+	private Node<T> tail;
+	private int size = 0;
+	
+	public SinglyLinkedList() {};
+	
+	public void findK(int k) {
```

Comment:
This is an example of a method that would benefit from a comment. As we don't know if findK finds the kTh element, the kTh from last element, the kth Biggest element etc.